Lab1/main.py

```diff
@@ -0,0 +1,16 @@
+from sortingmethods import quick_sort, insert_sort
+from random import randint
+
+def main():
+    array1 = [randint(-100, 100) for _ in range(0, 10)]
+    array2 = array1.copy()
```

Comment:
tu jeszcze okej, ale jakbyś miał więcej algorytmów do przetestowania to array1 array2 array3... tylko na potrzeby przechowywania kolejnej instancji tego samego jest bez sensu.